server/lib/payments.js

```diff
@@ -225,6 +225,8 @@ export const sendEmailNotifications = (order, transaction) => {
     sendOrderProcessingEmail(order);
     if (isProvider('opencollective.giftcard', order.paymentMethod)) {
       sendSupportEmailForManualIntervention(order); // async
+    } else {
```

Comment:
@Betree are we sure pledges are not falling in that case?

Reply:
I'm not, good catch. I'm going to test it manually

Reply:
I can confirm that it's safe, because we check `isActive` in `createOrder` (https://github.com/opencollective/opencollective-api/blob/e5d82f6a75838bb6cbcb7c3d74561e690544fbd7/server/graphql/v1/mutations/orders.js#L625) and never call `executeOrder` for pledged collectives